target/linux/ath79/dts/qca9558_tplink_archer-d7-v1.dts

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9558_tplink_archer-d7.dtsi"
+
+/ {
+	compatible = "tplink,archer-d7-v1", "qca,qca9558";
+	model = "TP-Link Archer D7 v1";
+};
+
+aliases {
+	label-mac-device = &wmac;
+};
```

Comment:
Aliases node belongs into the root node ( "/" node above).

This might explain the dts parsing issue reported ...

Reply:
@powo01